vsintegration/src/FSharp.Editor/Classification/ClassificationDefinitions.fs

```diff
@@ -24,33 +24,43 @@ module internal FSharpClassificationTypes =
     let [<Literal>] MutableVar = "FSharp.MutableVar"
     let [<Literal>] Printf = "FSharp.Printf"
-    let [<Literal>] ReferenceType = ClassificationTypeNames.ClassName
-    let [<Literal>] Module = ClassificationTypeNames.ModuleName
-    let [<Literal>] ValueType = ClassificationTypeNames.StructName
-    let [<Literal>] Keyword = ClassificationTypeNames.Keyword
-    let [<Literal>] Enum = ClassificationTypeNames.EnumName
-    let [<Literal>] Property = "FSharp.Property"
-    let [<Literal>] Interface = ClassificationTypeNames.InterfaceName
-    let [<Literal>] TypeArgument = ClassificationTypeNames.TypeParameterName
-    let [<Literal>] Operator = ClassificationTypeNames.Operator
     let [<Literal>] Disposable = "FSharp.Disposable"
 
     let getClassificationTypeName = function
-        | SemanticClassificationType.ReferenceType -> ReferenceType
-        | SemanticClassificationType.Module -> Module
-        | SemanticClassificationType.ValueType -> ValueType
-        | SemanticClassificationType.Function -> Function
+        | SemanticClassificationType.MutableRecordField
         | SemanticClassificationType.MutableVar -> MutableVar
         | SemanticClassificationType.Printf -> Printf
+        | SemanticClassificationType.DisposableValue
+        | SemanticClassificationType.DisposableType -> Disposable
+        | SemanticClassificationType.NameSpace -> ClassificationTypeNames.NamespaceName
+        | SemanticClassificationType.Exception
+        | SemanticClassificationType.Module
+        | SemanticClassificationType.Type
+        | SemanticClassificationType.TypeDef
+        | SemanticClassificationType.ReferenceType -> ClassificationTypeNames.ClassName
+        | SemanticClassificationType.ValueType -> ClassificationTypeNames.StructName
         | SemanticClassificationType.ComputationExpression
-        | SemanticClassificationType.IntrinsicFunction -> Keyword
+        | SemanticClassificationType.IntrinsicFunction -> ClassificationTypeNames.Keyword
         | SemanticClassificationType.UnionCase
-        | SemanticClassificationType.Enumeration -> Enum
-        | SemanticClassificationType.Property -> Property
-        | SemanticClassificationType.Interface -> Interface
-        | SemanticClassificationType.TypeArgument -> TypeArgument
-        | SemanticClassificationType.Operator -> Operator 
-        | SemanticClassificationType.Disposable -> Disposable
+        | SemanticClassificationType.Enumeration -> ClassificationTypeNames.EnumName
+        | SemanticClassificationType.Field
+        | SemanticClassificationType.UnionCaseField -> ClassificationTypeNames.FieldName
+        | SemanticClassificationType.Interface -> ClassificationTypeNames.InterfaceName
+        | SemanticClassificationType.TypeArgument -> ClassificationTypeNames.TypeParameterName
+        | SemanticClassificationType.Operator -> ClassificationTypeNames.Operator
+        | SemanticClassificationType.Constructor
```

Comment:
When the `Constructor` classification type is broken out into multiple, I may want to reconsider treating it as a method and instead go back to the previous behavior where it was treated as a Class or Struct based on the type being constructed